### PR TITLE
Adding submission checker

### PR DESCRIPTION
**Pull request by lqueipom**
Added:
- Added submission alert to course.html to notify whether submission was valid or invalid.
- Function validSubmission() to ratingScript.js which will be called before each attempt at submission.
    - If valid, it will proceed with submission as normally.
    - If not, it will prevent submission and ask the user to check the form fields. 

Modified:
- Removed label from checkbox in course.html as it is not necessary for this case.

### CODE REVIEW

File: src/main/webapp/ratingScript.js [hunk 1]
```diff
@@ -59,9 +59,11 @@ function addingAttributesToURL(ratingProperties) {
 }
 
 async function passRatingProperties() {
-  document.getElementById("retrieve-last-rating-message").innerHTML = "";
-  const urlAndData = await getRatingPropertiesToStore();
-  postRatingProperties(urlAndData[0], urlAndData[1]);
+  if (validSubmission() == true) {
```
Comment (ninaprabhu) [inline src/main/webapp/ratingScript.js:62]: if (validSubmission())

Comment (lqueipom) [reply src/main/webapp/ratingScript.js]: Thanks!!

**Comment by LukeGMTravis** [comment]
Starting review

File: src/main/webapp/ratingScript.js [hunk 2]
```diff
@@ -110,6 +112,34 @@ async function getLatestRating() {
...
+    document.getElementById("term-input"),
+    document.getElementById("prof-input"),
+    document.getElementById("rating-term"),
+    document.getElementById("rating-prof"),
+    document.getElementById("hours"),
+    document.getElementById("difficulty"),
+    document.getElementById("grade"),
+  ];
+
+  let i;
```
Comment (LukeGMTravis) [inline src/main/webapp/ratingScript.js:126]: You can declare this in the for loop

File: src/main/webapp/ratingScript.js [hunk 3]
```diff
@@ -110,6 +112,34 @@ async function getLatestRating() {
...
+  let successfulSubmissionMessage = document.getElementById(
+    "sucessful-submission-message"
+  );
+  for (i = 0; i < allFormFields.length; i++) {
+    // If any field is not field out properly.
+    if (!allFormFields[i].checkValidity()) {
+      successfulSubmissionMessage.innerHTML =
```
Comment (LukeGMTravis) [inline src/main/webapp/ratingScript.js:133]: Maybe "successfulSubmissionMessage" doesn't make as much sense here. Potential rename idea: "submissionStatusMessage"

File: src/main/webapp/ratingScript.js [hunk 4]
```diff
@@ -110,6 +112,34 @@ async function getLatestRating() {
   }
 }
 
+function validSubmission() {
```
Comment (LukeGMTravis) [inline src/main/webapp/ratingScript.js:115]: Nit: Usually functions returning a boolean like this have names like "isValidSubmission"

File: src/main/webapp/ratingScript.js (hunk 3 above)
Comment (LukeGMTravis) [inline src/main/webapp/ratingScript.js:133]: It's a little odd that a function returning a boolean indicating whether or not form fields are valid also has a side effect writing a message to the dom. Might want to pull out the side effect.

Comment (lqueipom) [reply src/main/webapp/ratingScript.js:133]: Makes sense, thanks for the comment making all the changes now!

**Pull request merged**